tests/org.eclipse.passage.lic.net.tests/META-INF/MANIFEST.MF

```diff
@@ -9,4 +9,5 @@ Bundle-Copyright: %Bundle-Copyright
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Fragment-Host: org.eclipse.passage.lic.net
 Require-Bundle: org.junit;bundle-version="4.12.0"
+Import-Package: org.junit.jupiter.api.function;version="5.8.1"
```

Comment:
Looks unused. Besides, migration to Jupiter is out of the issue scope.

Reply:
Only the `ThrowingSupplier` interface from that package was used. JUnit-4 does not offer such interface.
However with the rework/introduction of the `TestKeyKeeperFactory` such interface is not necessary anymore.